### ISSUE TITLE
Ability to write moar buffer to a file or to stdout

### ISSUE THREAD

**Issue by koraa** [walles/moar#135]
Hi thank you for writing this awesome tool!

I propose adding the ability to write the buffer to a file or stdout; this allows me not to have to think about whether I need to do do something with the output of a command and would greatly facilitate use cases such as paging every command through moar by default as a shell feature.

**Comment by walles** [comment]
You can already do `echo hello | moar > hello.txt`.

Is this what you want to do?

If not, an example of how you'd like this to work would help!

**Comment by koraa** [comment]
No, I am well aware of redirection. Its more like:

```
$ some_really_expensive_command | moar
$ some_really_expensive_command > myfile.txt # Shit I really should have written this to a file
```

**Comment by walles** [comment]
Right, so once inside of `moar`, you'd like to press some button, which would make `moar` save its buffer contents to a file?

**Comment by koraa** [comment]
Precisely.

**Comment by walles** [comment]
The title here says "or to stdout".

I can see the use case for sending to a file, but what about `stdout`?

Let's say I only did to file and not to `stdout`. Wouldn't that solve the whole use case of `Shit I really should have written this to a file`?

**Comment by koraa** [comment]
There would be some utility in user-in-the-loop data processing pipelines; opening multiple files and deciding which ones to process. Although if you write to a file, you can always write to "/dev/stdout"

**Comment by walles** [comment]
Nowadays you can press v inside of moar to open the buffer in an editor.

Then from that editor you can save into a file of your choice.

Not exactly what you are asking for, but it should be of some help at least.

I use it for editing psql output, so for me this is super useful!

**Comment by walles** [comment]
Closing this since I believe `v` should be good enough, and for lack of updates.
